### PR TITLE
Fix bugs in mule test stage

### PR DESCRIPTION
**Pull request by btoll**
## Summary

- fix status file name
- change name of image in mule yaml
- add vars to dynamically account for both algorand and algorand-devtools packages (all channels)

## Test Plan

`VERSION=2.2.0 BRANCH=rel/beta CHANNEL=beta mule -f package-test.yaml package-test`
`VERSION=2.1.6 BRANCH=rel/stable CHANNEL=stable NETWORK=mainnet SHA=df65da2b mule -f package-test.yaml package-test`

### CODE REVIEW

File: scripts/release/mule/test/test.sh [hunk 1]
```diff
@@ -16,9 +16,15 @@ export CHANNEL=${CHANNEL:-$(./scripts/compute_branch_channel.sh "$BRANCH")}
 export NETWORK=${NETWORK:-$(./scripts/compute_branch_network.sh "$BRANCH")}
 export SHA=${SHA:-$(git rev-parse HEAD)}
 export VERSION=${VERSION:-$(./scripts/compute_build_number.sh -f)}
+ALGORAND_PACKAGE_NAME=$([ "$CHANNEL" = beta ] && echo algorand-beta || echo algorand)
+DEVTOOLS_PACKAGE_NAME=$([ "$CHANNEL" = beta ] && echo algorand-devtools-beta || echo algorand-devtools)
+export ALGORAND_PACKAGE_NAME
+export DEVTOOLS_PACKAGE_NAME
 
+# Note that currently we are only supporting amd64 packages (20201030).
```
Comment (egieseke) [inline scripts/release/mule/test/test.sh:24]: Currently only supporting **testing** of amd64...

File: scripts/release/mule/test/test.sh [hunk 2]
```diff
@@ -16,9 +16,15 @@ export CHANNEL=${CHANNEL:-$(./scripts/compute_branch_channel.sh "$BRANCH")}
 export NETWORK=${NETWORK:-$(./scripts/compute_branch_network.sh "$BRANCH")}
 export SHA=${SHA:-$(git rev-parse HEAD)}
 export VERSION=${VERSION:-$(./scripts/compute_build_number.sh -f)}
+ALGORAND_PACKAGE_NAME=$([ "$CHANNEL" = beta ] && echo algorand-beta || echo algorand)
```
Comment (onetechnical) [inline scripts/release/mule/test/test.sh:19]: Maybe expand this from one liners, and add algorand-nightly support?

File: scripts/release/mule/test/test.sh (hunk 1 above)
Comment (onetechnical) [inline scripts/release/mule/test/test.sh:24]: Do we need this restriction? I think we're packaging all architectures now, so it's feasible to test all.

File: scripts/release/mule/test/tests/pre/verify_control_files.sh [hunk 3]
```diff
@@ -15,41 +15,43 @@ then
...
+    echo "[$0] The dependency for $ALGORAND_PACKAGE_NAME version $VERSION is correct."
 else
     # Note that the .spec file isn't packaged in the RPM. There are tools such `rpmrebuild` that
     # attempt to generate the .spec file, but it doesn't give us the info we need.
     #
     # Instead, we'll just install using `dpkg` and grep the error stream.
-    if ! rpm -i "./tmp/node_pkgs/$OS_TYPE/$ARCH_TYPE/algorand-devtools-$VERSION"*"$ARCH_BIT".rpm 2> "$RPMTMP/rpm.install"
+    #
+    # Also, note that the RPM package DOES NOT have the CHANNEL in its filename (unlike DEB)!!
+    if ! rpm -i "./tmp/node_pkgs/$OS_TYPE/$ARCH_TYPE/algorand-devtools-"*"$VERSION"*"$ARCH_BIT".rpm 2> "$RPMTMP/rpm.install"
```
Comment (egieseke) [inline scripts/release/mule/test/tests/pre/verify_control_files.sh:37]: Is ARCH_BIT correct?

**Review by egieseke — APPROVED** [review]
Looks OK.

File: scripts/release/mule/test/tests/pre/verify_control_files.sh (hunk 3 above)
Comment (onetechnical) [reply scripts/release/mule/test/tests/pre/verify_control_files.sh:37]: I think this is correct for RPM

File: scripts/release/mule/test/test.sh (hunk 1 above)
Comment (btoll) [reply scripts/release/mule/test/test.sh:24]: Yes, that's true, I was thinking that we only support amd64 from our repos.  I'll remove this.

File: scripts/release/mule/test/test.sh (hunk 2 above)
Comment (btoll) [reply scripts/release/mule/test/test.sh:19]: One-liners rule!

Are we officially supporting nightlies now?  I thought that was still being determined.

File: scripts/release/mule/test/tests/pre/verify_control_files.sh (hunk 3 above)
Comment (btoll) [reply scripts/release/mule/test/tests/pre/verify_control_files.sh:37]: Yes, and it's been tested.

File: scripts/release/mule/test/test.sh (hunk 1 above)
Comment (btoll) [reply scripts/release/mule/test/test.sh:24]: I'm going to remove this.

File: scripts/release/mule/test/test.sh (hunk 2 above)
Comment (onetechnical) [reply scripts/release/mule/test/test.sh:19]: Hmm, maybe not for packaging. Something to look at in the future.

**Review by onetechnical — APPROVED** [review]

**Review by egieseke — APPROVED** [review]

**Pull request merged**